planet/clients/subscriptions.py

```diff
@@ -46,22 +46,17 @@ async def list_subscriptions(self,
             ClientError
 
         """
-        # Temporary marker for behavior of module with unpatched state.
-        if _fake_subs is None:
-            raise NotImplementedError
-
-        if status:
+        try:
             select_subs = (dict(**sub, id=sub_id) for sub_id,
                            sub in _fake_subs.items()
-                           if sub['status'] in status)
-        else:
-            select_subs = (
-                dict(**sub, id=sub_id) for sub_id, sub in _fake_subs.items())
+                           if not status or sub['status'] in status)
```

Comment:
I turned if/else blocks into one statement. This is more like the production code will be and works better with Python's coverage module.